src/frontend/cypress/Frank/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	>
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.ibissource</groupId>
+	<artifactId>frank-flow-e2e-frank</artifactId>
+	<version>1.0</version>
+	<packaging>war</packaging>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.ibissource</groupId>
+			<artifactId>ibis-adapterframework-webapp</artifactId>
+			<version>[7.6,)</version>
+			<type>war</type>
+		</dependency>
+		<dependency>
+			<groupId>org.ibissource</groupId>
+			<artifactId>frank-flow</artifactId>
+			<version>[1.0,)</version>
```

Comment:
Is this the right version?

Do you need the frank-flow in this pom? If it is needed, it is probably better to use the current version: 2.0.1-SNAPSHOT

Reply:
Er staat `<version>[1.0,)</version>`. Dat is niet een vaste versie, maar betekent dat de nieuwste versie gebruikt moet worden die groter of gelijk is aan 1.0. Zie https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402.

Reply:
Oke, ik wist niet zeker of dit ook major versions bevatte, maar lijkt het wel op.